### PR TITLE
Template Style for the Header/App/Workflow.js

### PR DESCRIPTION
**Pull request by MinhLu12**
@trilxs Since you're coming up with prototypes, I'd start making them with this CSS I've made and eventually with the finalized version.

You can pull from this branch to see what it looks like but here are screen shots:

capture

**NOTE:**
1) It's resizeable
2) The tabs glow up when you hover above them
3) The rest of the app is not within a container

**UPDATE:**
Here's how the workflow looks like (still needs work):
capture

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" style="background-color: #E6ECF0">
   <head>
```
Comment (MinhLu12) [inline client/public/index.html:3]: Make the whole application this color

File: client/src/components/App/App.js [hunk 2]
```diff
@@ -37,13 +37,14 @@ class App extends Component {
...
-                <Router history={history}>
-                    <div>
-                        {/* Do not show the header when we're in the errors page */}
-                        {this.state.hideHeader ? null : <Header />}
+            <Router history={history}>
+                <div>
+                    {/* Do not show the header when we're in the errors page */}
+                    {this.state.hideHeader ? null : <Header />}
+                    <div className="container">
                         <Switch>
```
Comment (MinhLu12) [inline client/src/components/App/App.js]: We do not want our Header in the container (of 85% width), so take it out.

File: client/src/components/Header/Header.css [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    }
+
+    /* All links in the header have this style */
+    nav ul li a {
+        padding: 0px 15px;
+        text-decoration:none;
+        transition: all 0.3s ease-in-out;
+        -webkit-transition: all 0.3s ease-in-out;
+        -moz-transition: all 0.3s ease-in-out;
+    }
```
Comment (MinhLu12) [inline client/src/components/Header/Header.css:32]: All the CSS thus far for the Header. It uses specificity but probably not the most optimal?

File: client/src/styles/global.css [hunk 4]
```diff
@@ -2,4 +2,10 @@
...
 :global(.mb-0) { margin-bottom: 0px; }
-:global(.ml-0) { margin-left: 0px; }
\ No newline at end of file
+:global(.ml-0) { margin-left: 0px; }
+
+/* Padding 20% */
+:global(.pt-20p) { padding-top: 20% }
+:global(.pr-20p) { padding-right: 20% }
+:global(.pb-20p) { padding-bottom: 20% }
+:global(.pl-20p) { padding-left: 20% }
```
Comment (MinhLu12) [inline client/src/styles/global.css]: Adding more to our global styles since padding of 20% might be used a lot(?)

**Review by yaoxiaotong — APPROVED** [review]
not knowing much react yet, but this looks good.

**Pull request merged**